R_scripts/licordat.R

```diff
@@ -14,15 +14,27 @@ read_licor_data <- function(filename) {
   r2 <- file[grepl("^Lin_R2:", file)]
   nobs <- length(file[grepl("^Obs#:", file)])
   date <- file[which(grepl("^Type", file)) + 1]
-  temp <- file[grepl("^Comments:", file)]
+  temp20 <- file[grepl("^Comments:", file)]
+
+  tablestarts <- grep("^Type", file)
+  tablestops <- grep("^CrvFitStatus", file)
+  tcham <- matrix()
+  t5 <- matrix()
+  
+  for (i in 1:length(tablestarts)) {
+    df <- read.table(filename, skip = tablestarts[i] - 1, header = TRUE, 
+               nrows = tablestops[i] - tablestarts[i] - 1)
+    tcham[i] <- round(mean(df$Pressure), digits = 2)
```

Comment:
This is confusing. Are the column headers off by 1 or something? Why are we reading Pressure and LATITUDE? At the very least this needs an explanatory comment.